docs/installation.rst

```diff
@@ -125,6 +128,9 @@ The installation of the Connectome Mapper 3, including `cmpbidsappmanager`, cons
 
         $ conda env create -f /path/to/downloaded/conda/environment[_macosx].yml
 
+    .. note:: This can take a little bit of time depending on your connection speed and your machine.
+              You should also expect that it would take a space of around 2.8GB on your machine.
```

Comment:
For consistent wording I would say "...it would take around 2.8GB of space on your machine"

Reply:
```suggestion
              it should take around 2.8GB of space on your machine.
```